### DISCUSSION TITLE
Uninstallation procedure

### DISCUSSION THREAD

**Discussion by steve10883** [pilotmoon/Scroll-Reverser#166] [Q&A]
Hi,

I have uninstalled Scroll Reverser and now every time I boot up I receive the following error message:

_There is no application set to open the URL x-pilotmoon-scroll-reverser://launch._

What are the full uninstallation instructions for this app?

Thanks.

**Comment by pilotmoon** [comment]
Try installing it again, then uncheck the start at login option, and then delete it again.

**Comment by pilotmoon** [comment]
This from another user, re. removing "allow in background items", may help:

- `sudo sfltool dumpbtm` : gave me a full list of login items, including "Allow in background" ones.
- Your application seems to have been affected by App Translatocation making it difficult to locate (unexisting folder, etc...especially that I went out of my way to clean temp files, not sure about the details about this security feature)
-  Anyway, I nuked the whole Login items list with `sudo sfltool resetbtm`, rebooted again, it basically forced macOS to recheck what is actually in LaunchAgents, LaunchDaemons, etc... folders.
- Result: it picked up all the applications that still had a proper .plist to appear, cleared Scroll Reverser. sudo sfltool dumpbtm now shows a clean list.

**Reply by varungupta3009** [reply]
This helped. Thanks a lot! :D

**Reply by pilotmoon** [reply]
Awesome. Note this should only be needed for versions of PopClip before v1.9. Newest version uses a different launch mechanism.
